commands/operator-sdk/cmd/new.go

```diff
@@ -161,6 +161,11 @@ func execCmd(stdout *os.File, cmd string, args ...string) {
 }
 
 func pullDep() {
+	depPath, err := exec.LookPath(dep)
+	if err != nil {
+		cmdError.ExitWithError(cmdError.ExitError, err)
+	}
+	fmt.Printf("dep is available at %s\n", depPath)
```

Comment:
probably no need to add `fmt.Printf("dep is available at %s\n", depPath)` since we know that if dep is not there we will error out.